experiments/utils/download.py

```diff
@@ -79,9 +107,10 @@ def create_file_response_single(file_format: str, data_point: DataPoint) -> \
 
     # Set the content disposition header, so that browser see the page as a
     # downloadable file
-    filename = "{}-{}.{}".format(
+    filename = "{}-{}-{}.{}".format(
         data_point.experiment.title,
-        data_point.session.subject_id,
+        str(data_point.session.subject_id).zfill(DEFAULT_ZFILL),
+        str(data_point.number).zfill(DEFAULT_ZFILL),
         file_format
```

Comment:
Why not use the new ``_create_file_name`` here as well?

Reply:
Well in the zip, the files always had a slightly different name compared to downloading one single datapoint. They are missing the experiment.title. So I figured this was intended. I can adapt the _create_file_name function to take an experiment_name parameter and use that as well. Do you want the file in the zip to have the experiment name included as well?

Reply:
So that is added in https://github.com/UiL-OTS-labs/web-experiment-datastore/pull/39/commits/f433da467646f200c4594e5621d198a9768454b1
I used the experiment 'Gewoon iets proberen' I also remove the space to "-" because I feel that makes file better readable and converted to lower case letters. Perhaps we want to give the final .zip file a similar name.